detekt.yml

```diff
@@ -84,7 +84,7 @@ formatting:
     active: true
     autoCorrect: true
   ExpressionBodySyntaxLineBreaks:
-      active: true
+      active: false
```

Comment:
Could enable it again, but detekt was not allowing me to compile in any way (tried many workarounds) because of format requirement collisions on 2 different rules (I think it was this one and MaxLineLength). I didn't have any other choice than disabling this.